cosmo_tester/test_suites/bootstrap_based_tests/multi_network_test.py

```diff
@@ -134,20 +128,34 @@ def multi_network_hello_worlds(cfy, managers, attributes, ssh_key, tmpdir,
         hello.cleanup()
 
 
-def _enable_nics(manager, networks, tmpdir, logger):
-    """ Extra network interfaces need to be manually enabled on the manager """
+def _enable_nics(manager):
```

Comment:
btw this should maybe probably be a method on the manager? it's one of the few (only?) real consumer of `.networks`, and all it does has to do with accessing attributes on the manager